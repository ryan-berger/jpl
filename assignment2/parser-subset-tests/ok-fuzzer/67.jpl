time let a1Z. = .92
show .0852
write image I5M.5J_5O_2R_() to ";"
return .42
time print "[."
read image "" to h
read image "v2" to P
