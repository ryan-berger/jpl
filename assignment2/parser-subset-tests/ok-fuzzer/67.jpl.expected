(TimeCmd (StmtCmd (LetStmt (ArgLValue (VarArgument a1Z.)) (FloatExpr 0))))
(ShowCmd (FloatExpr 0))
(WriteImageCmd (CallExpr I5M.5J_5O_2R_) ";")
(StmtCmd (ReturnStmt (FloatExpr 0)))
(TimeCmd (PrintCmd "[."))
(ReadImageCmd "" (VarArgument h))
(ReadImageCmd "v2" (VarArgument P))
